device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -796,6 +796,11 @@ void setConnectionType(IoTHubConnectionType connectionType)
      *	      in case of MQTT and AMQP protocols, this option specifies the interval in milliseconds
      *	      between spawning a thread that dequeues a message from the SDK's queue of received messages.
      *
+     *	    - <b>SetMaxMessagesSentPerThread</b> - this option is applicable to all protocols.
```

Comment:
is there a recommendation for these values that we can provide to the user; eg. the effects of setting a very large value or a very low value?

Reply:
It'll depend too much on how the customer uses the SDK. I can try to add a bit more context here, though.